### ISSUE TITLE
Implement DMA

### ISSUE THREAD

**Issue by PizzaAllTheWay** [vortexntnu/vortex-acoustics#102]
Make DMA work parallel while CPU on MCU is doing other work.
Make sure that we can read values from DMA while DMA is doing its own thing, this will be a challenge.

**Comment by PizzaAllTheWay** [comment]
No need for that since new ADC code uses interrupts to sample all the samples fast enough without loosing any data. Felix is the big bouss mastermind hehe :D

**Comment by theBadMusician** [comment]
I do not think this issue should be closed, as this solution may need to be looked into, in case we need faster or more robust sampling.

Reopening this issue, and adding "future" label.

**Comment by PizzaAllTheWay** [comment]
oki
